jujugui/static/gui/src/app/components/inspector-config/inspector-config.js

```diff
@@ -231,6 +232,7 @@ YUI.add('inspector-config', function() {
       var configElements = [];
 
       Object.keys(charmOptions).forEach((key) => {
+        var disabled = this.props.acl.isReadOnly();
```

Comment:
This doesn't need to be inside the loop as this bool is global. Save some computation here.